packages/ListBox/src/store/Provider.js

```diff
@@ -10,26 +9,19 @@ function initializeState(props) {
   const {
     childrenOptions,
     height,
-    isDisabled,
-    isInline,
     isMulti,
     isOpen,
-    placeholder,
     refFilterInput,
     refFooterContainer,
     refListBox,
     refListBoxContainer,
     refTrigger,
     refTriggerContainer,
     refLabel,
```

Comment:
I considered moving `refLabel` to `PropsProvider` too, but since it's passed by reference, not value, any subsequent update by the consumer is reflected in the "store" so I followed the "if it ain't broke, don't fix it" policy.